README.md

```diff
@@ -1,51 +1,161 @@
-# Bare Metal Blinky
+![](./resources/official_armmbed_example_badge.png)
+# Bare metal blinky Mbed OS example
 
-This guide reviews the steps required to get Blinky with Bare Metal working on an Mbed OS platform.
+This example shows how to achieve memory optimisations in Mbed. Starting with a simple blinky application, the example illustrates how to enable the bare metal profile and further memory optimisations.
 
-Please install [mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+You can build this project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
 
-## Import the example application
+## Application functionality
 
-From the command-line, import the example:
+The `main()` function toggles the state of a digital output connected to an LED on the board.
 
+## Prerequisites
+
```

Comment:
Can we keep the generic parts of the README the same please https://github.com/ARMmbed/mbed-os-example-blinky/blob/master/README.md